src/Common/Doctrine/Entity/Meta.php

```diff
@@ -79,6 +79,20 @@ class Meta
      */
     private $urlOverwrite;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255, name="canonical_url", nullable=true)
+     */
+    private $canonicalUrl;
```

Comment:
this is only allowed in version 6 because this is a database change. You can always implement it in the metadata and take it out of there upon serialisation.
Otherwise you need to change the base of the PR to restructure-core

Reply:
This is changed. When this is merged into master I will create a new PR to restructure-core.